### PR TITLE
check custom corp count

### LINKED ISSUE #2789: Game creation with less than (players * corpsPerPlayer) corporations should be an error
The game creation interface should know deterministically how many corporations will be in play, and if that's not enough to give everyone the desired number, seems like that should be an error instead of reverting to 2 corps per player.

image

### PR DESCRIPTION
**Pull request by vsrisuknimit**
This PR can close #2789 .

### CODE REVIEW

File: src/components/CreateGameForm.ts [hunk 1]
```diff
@@ -424,6 +425,16 @@ export const CreateGameForm = Vue.component('create-game-form', {
         }
       }
 
+      // Check custom corp count
+      if (customCorporationsList.length > 0) {
+        const neededCorpsCount = players.length * startingCorporations;
+
+        if (customCorporationsList.length < neededCorpsCount) {
+          window.alert('Must select at least ' + neededCorpsCount + ' corporations');
```
Comment (alrusdi) [inline src/components/CreateGameForm.ts:433]: Can you please add i18n?

**Review by alrusdi — CHANGES_REQUESTED** [review]

File: src/components/CreateGameForm.ts (hunk 1 above)
Comment (vsrisuknimit) [reply src/components/CreateGameForm.ts]: added

File: src/components/CreateGameForm.ts [hunk 2]
```diff
@@ -430,7 +431,7 @@ export const CreateGameForm = Vue.component('create-game-form', {
         const neededCorpsCount = players.length * startingCorporations;
 
         if (customCorporationsList.length < neededCorpsCount) {
-          window.alert('Must select at least ' + neededCorpsCount + ' corporations');
+          window.alert($t('Must select at least ') + neededCorpsCount + $t(' corporations'));
```
Comment (alrusdi) [inline src/components/CreateGameForm.ts:434]: I'm sorry it is little bit incorrect. You should pass a Message object to this function with [neededCorpsCount] as data and something like "Must select at least ${0} colonies" as message

Comment (vsrisuknimit) [reply src/components/CreateGameForm.ts:434]: @alrusdi I don't follow. Can you do it in another PR or point me to a line that does something similar? None of the alert message in this file use v-18n so far.

Comment (alrusdi) [reply src/components/CreateGameForm.ts:434]: Can you try something like this:
```$t({type: LogMessageDataType.RAW_STRING, data: [neededCorpsCount] message: 'Must select at least ${0} colonies'} as Message)```
?

Comment (vsrisuknimit) [reply src/components/CreateGameForm.ts:434]: image

Comment (vsrisuknimit) [reply src/components/CreateGameForm.ts:434]: @alrusdi would you like to try it later in a different PR?

**Review by alrusdi — APPROVED** [review]
ok, let's do i18n later

**Pull request merged**